### PR TITLE
Add known docker workarounds page

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

File: content/docs/distributing-an-application/known-docker-workarounds.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+date: "2018-01-09T20:02:20Z"
+title: "Known Docker Installation Issues and Workarounds"
+description: "Issues that Replicated has run into with various Docker installations"
+keywords: "installing, docker"
+weight: "302"
+categories: [ "Distributing an Application" ]
```
Comment (emosbaugh) [inline content/docs/distributing-an-application/known-docker-workarounds.md]: maybe this should be a kb article?

File: content/docs/distributing-an-application/known-docker-workarounds.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+index: "docs"
+tags: ["Installing Replicated"]
+---
+
+## Overlay driver enabled despite being on an unsupported configuration
+The default driver chosen by some Docker versions on RHEL and CentOS, overlay, has requirements that are not always satisfied. Overlay and Overlay2 require ftype=1 and a kernel >= 3.18, which not all installations fulfill. View the [Docker documentation](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/#prerequisites) for details.
```
Comment (marccampbell) [inline content/docs/distributing-an-application/known-docker-workarounds.md]: can we add recommendations here?

File: content/docs/distributing-an-application/installing-manually.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ If you choose not to run the [installation script](/docs/distributing-an-applica
 To manually install start by checking you are running on a support operating system and follow the 4 steps.
 
 ### 1. Install Docker
-We recommend Docker version {{< docker_version_default >}}.  Refer to the Docker Installation Guide for [Debian](https://docs.docker.com/engine/installation/linux/debian/), [Ubuntu](https://docs.docker.com/engine/installation/linux/ubuntulinux/), [CentOS](https://docs.docker.com/engine/installation/linux/centos/), [Fedora](https://docs.docker.com/engine/installation/linux/fedora/), or [RHEL](https://docs.docker.com/engine/installation/linux/rhel/).
+We recommend Docker version {{< docker_version_default >}}.  Refer to the Docker Installation Guide for [Debian](https://docs.docker.com/engine/installation/linux/debian/), [Ubuntu](https://docs.docker.com/engine/installation/linux/ubuntulinux/), [CentOS](https://docs.docker.com/engine/installation/linux/centos/), [Fedora](https://docs.docker.com/engine/installation/linux/fedora/), or [RHEL](https://docs.docker.com/engine/installation/linux/rhel/). Please note [these issues](/docs/distributing-an-application/known-docker-workarounds/) we have had with installing docker.
```
Comment (marccampbell) [inline content/docs/distributing-an-application/installing-manually.md]: Change the last sentence to:

Some environments may require additional configuration to install Docker with a production-ready and stable configuration.

File: content/docs/distributing-an-application/known-docker-workarounds.md (hunk 1 above)
Comment (smithers1221) [reply content/docs/distributing-an-application/known-docker-workarounds.md]: Minor edit suggestions:

On first launch, there will be no "Start Now" button while Replicated is still pulling application images. The button will appear after the pull is finished.

When a new node running Replicated Operator joins the cluster, Replicated will want to run preflight checks on it before starting the application. In this case, the "Start Now" button will be replaced with the "Run Checks" button.

**Pull request merged**